### DISCUSSION TITLE
supabase.auth.signOut({ scope: 'local' }) behaves as 'global' by default

### DISCUSSION THREAD

**Discussion by captainahren** [supabase/supabase#31017] [Questions]
I encountered an issue with the supabase.auth.signOut method in the latest version of @supabase/supabase-js. 
When I call the method with { scope: 'local' }, it behaves as if the scope is set to 'global' by default. It seems to clear the server-side session instead of just the local session.

await supabase.auth.signOut({ scope: 'local' });

**Comment by GaryAustin1** [comment]
Try options:{scope:’local’}
